near-sdk/src/collections/lazy_option.rs

```diff
@@ -17,7 +17,7 @@ const ERR_VALUE_DESERIALIZATION: &str = "Cannot deserialize value with Borsh";
 #[derive(BorshSerialize, BorshDeserialize)]
 pub struct LazyOption<T> {
     storage_key: Vec<u8>,
-    #[borsh_skip]
+    #[borsh(skip)]
     el: PhantomData<T>,
```

Comment:
`PhantomData` is hardcoded [exception](https://github.com/near/borsh-rs/blob/master/borsh-derive/src/internals/generics.rs#L238), when type parameter is skipped from being added to any kinds of bounds.